R/utils.R

```diff
@@ -125,3 +125,25 @@ include_js_files <- function(pattern) {
   )
   return(singleton(lapply(js_files, includeScript)))
 }
+
+
+#' Assert logical expression
+#'
+#' Assert logical expression. If object is not a logical expression then error is thrown.
+#' @param x anything
+#' @return NULL or error
+#' @keywords internal
+assert_logical_expr <-  function(x) {
```

Comment:
Perhaps add a return value of `invisible(x) to conform to `checkmate::assert_*` functions?